roles/openshift_logging/templates/kibana.j2

```diff
@@ -44,15 +44,19 @@ spec:
 {% if kibana_cpu_limit is not none %}
               cpu: "{{kibana_cpu_limit}}"
 {% endif %}
-{% if kibana_memory_limit is not none %}
               memory: "{{kibana_memory_limit}}"
```

Comment:
Should we also have a default value specified here since we don't have one when this template is used (e.g. what if a user specifies the value as null)